src/client/home/MintSection/MintingModalButton/MintingModal/MintButton/MintButton.tsx

```diff
@@ -31,21 +31,12 @@ const MintButton: React.FC<MintButtonProps> = ({ numberOfItems, className }) =>
     <DisplayIf
       condition={isEligable}
       falsy={
-        <Button
-          onClick={() => {
-            publishEvent(EngageEventTypes.ClickedIneligibleForMint, {
-              walletAddress: address,
-            });
-          }}
-          className={`w-full border border-gray-500 p-2 bg-gray-600 text-white !rounded-md ${className}`}
-        >
-          {t('Ineligible For Mint')}
-        </Button>
+        <IneligibleMintForm/>
       }
     >
-      <Button onClick={handleOnClick} className={`w-full border border-gray-500 p-2 bg-gray-900 text-white !rounded-md ${className}`}>
+      <PrimaryButton onClick={handleOnClick} className={`w-full border border-gray-500 p-2 bg-gray-900 text-white !rounded-md ${className}`}>
```

Comment:
create a dedicated button for this or remove the className

Also, please avoid `!` in the css